apps/chat/src/components/Promptbar/components/Prompt.tsx

```diff
@@ -423,14 +430,14 @@ export const PromptComponent = ({
             isHighlighted={isHighlited}
             featureType={FeatureType.Prompt}
             containerClassName={classNames(
-              isSelectMode && !isExternal && 'group-hover/prompt-item:hidden',
+              isSelectMode && !isExternal && 'group-hover:hidden',
```

Comment:
Are you sure that hover logic isn't broken?

Reply:
I checked and I don't think it's breaking anything.
BG of `<ShareIcon />` was broken because of the different groups.